### DISCUSSION TITLE
Ruby and PHP Libraries

### DISCUSSION THREAD

**Discussion by ankane** [rdkit/rdkit#7703] [Show and tell]
Hi, I wanted to share there are now [Ruby](https://github.com/ankane/rdkit-ruby) and [PHP](https://github.com/ankane/rdkit-php) bindings for RDKit :tada:

They use the CFFI interface introduced in [this post](https://greglandrum.github.io/rdkit-blog/posts/2021-05-01-rdkit-cffi-part1.html).

If you have any feedback, let me know or feel free to create an issue in either of the projects. Thanks!

**Comment by greglandrum** [comment]
Very cool. Thanks for doing this @ankane
